feat(readme): update readMe, update replace method (#23)
akveo/eva-icons: README.md

## Documentation

### `eva.replace(options)`

Replaces all elements that have a `data-eva` attribute with SVG markup.

`options` optional object.

#### Available 'option' properties:
| Name |  Type   |  Default value | Description |
|------| ------  | -------------  |-------------|
| fill | string | none           | Icon color  |
| width | string or number | 24px    | Icon width  |
| height | string or number | 24px    | Icon height  |
| class | string | none | Custom css class  |
| animation | object | none    | [Icon animation](#animation)  |

### Animation 
- Add the `data-eva-animation` attribute with the animation type `(zoom, pulse, shake and flip)` to an element:

```html
<i data-eva="github" data-eva-animation="zoom"></i>
```

> - By default animation works by hover and has infinity `false`, if you want change this behavior add `data-eva-hover="true | false"` and `data-eva-infinite="true | false"`

Reviewer: this sentence is quite hard to understand, try to rephrase

```html
<i data-eva="github" data-eva-animation="zoom" data-eva-hover="false" data-eva-infinite="true"></i>
```

> **Note:** In the above example github icon will be always animated.

- Pass animation as property in a `eva.replace` method.

```js
eva.replace({
  aniamtion: {
    type: string, // zoom, pulse, shake, flip
    hover: booleam, // default true
    infinite: boolean, // default false
  }
});
```
> **Note:** The animation will be applied to all replaced elements.

> - Add class `eva-parent-hover` to parent container if you want that the animation works by hover on the parent element.

Reviewer: ```suggestion
- Add `eva-parent-hover` class to the parent container in a case you want to activate the animation hovering on the parent element.
```

## Documentation

### `eva.replace(options)`

Replaces all elements that have a `data-eva` attribute with SVG markup.

`options` optional object.

#### Available 'option' properties:
| Name |  Type   |  Default value | Description |
|------| ------  | -------------  |-------------|
| fill | string | none           | Icon color  |
| width | string or number | 24px    | Icon width  |
| height | string or number | 24px    | Icon height  |
| class | string | none | Custom css class  |
| animation | object | none    | [Icon animation](#animation)  |

### Animation 
- Add the `data-eva-animation` attribute with the animation type `(zoom, pulse, shake and flip)` to an element:

```html
<i data-eva="github" data-eva-animation="zoom"></i>
```

- Additional animation attributes:

>   * `data-eva-hover`: Makes an animation available by hover. Default value is `true`. Available true or false.

Reviewer: ```suggestion
  * `data-eva-hover`: Makes the animation available on hover. Default value is `true`. Available `true` or `false`.
```

Author: done

>   * `data-eva-infinite`: Makes animation as infinity. Default value is `false`. Available true or false.

Reviewer: ```suggestion
  * `data-eva-infinite`: Makes the animation infinite. Default value is `false`. Available `true` or `false`.
```

Author: done
